src/strategy/saml_client.ts

```diff
@@ -70,23 +72,32 @@ export class CustomSamlClient extends PassportSaml.SAML {
   public generateAuthorizeRequest(
     req: express.Request,
     isPassive: boolean,
+    isHttpPostBinding: boolean,
```

Comment:
This is a change that impacts the API of the module and may lead to eventual misuses.

I see is due to a change occurred in `passport-saml` module, more specifically [here](https://github.com/node-saml/passport-saml/pull/207). As far as I get it, the newly-introduced `isHttpPostBinding` option ha no effect when falsy (see [here](https://github.com/richardTowers/passport-saml/blob/6bd4754336b17dd7396f878acf6a851ec1f9d9ab/lib/passport-saml/saml.js#L217)).

For the sake of keeping this PR in scope, I'd move such change and related ones on a dedicate PR on which we can discuss it properly. When calling `super.generateAuthorizeRequest`, we may just pass `false`.

Reply:
More on that: we may actually need to move to [`passport-saml@3`](https://github.com/pagopa/io-spid-commons/pull/89).

Reply:
Replaced by PR #91